docs/src/lib/sets/SimpleSparsePolynomialZonotope.md

```diff
@@ -0,0 +1,11 @@
+```@meta
+CurrentModule = LazySets
+```
+
+# [SimpleSparsePolynomialZonotope](@id def_SimpleSparsePolynomialZonotope)
+
+```@docs
+SimpleSparsePolynomialZonotope
+nparams
+expmat
```

Comment:
The downside with only including docs for those methods and not for the others is that the page `SimpleSparsePolynomialZonotope` will not display most common methods that apply to it. There is no easy way to automate the behavior we'd like apart from adding those manually at the expense of some repetition (which is what we've currently been doing).

Reply:
So should I include also genmat order etc.?

Reply:
Why do you get a repetition? You can just add the type annotation (see an example [here](https://github.com/JuliaReach/LazySets.jl/blob/master/docs/src/lib/sets/Zonotope.md)).

Reply:
I think Marcelo meant repetition as "you have to type it manually from every set". I don't mind the typing but fwiw I do think the current system (one docstring for each method even if they are all conceptually identical) makes docstrings very user-unfriendly if used from the REPL (which is where I use them generally, don't know if it's more common to look at docstrings from the docs or REPL)

Reply:
> So should I include also genmat order etc.?

I would prefer that, yes. Otherwise there is no point in writing this documentation :)